### ISSUE TITLE
Exception: Cannot enlarge memory arrays, if the font file size > 10M

### ISSUE THREAD

**Issue by Vkrun** [intrigus/gdx-freetype-gwt#5]
Hi, Thanks for your work for this project. I use it on my web game, It's just works fine before.
When I use a new ttf file on my game, I got an error:

> Error: java.lang.RuntimeException: com.google.gwt.core.client.JavaScriptException: (String) : abort("Cannot enlarge memory arrays. Either (1) compile with  -s TOTAL_MEMORY=X  with X higher than the current value 16777216, (2) compile with  -s ALLOW_MEMORY_GROWTH=1  which adjusts the size at runtime but prevents some optimizations, (3) set Module.TOTAL_MEMORY to a higher value before the program runs, or if you want malloc to return NULL (0) instead of this abort, compile with  -s ABORTING_MALLOC=0 ") at jsStackTrace@http://localhost:8080/dist/html/freetype.js:1:19835 stackTrace@http://localhost:8080/dist/html/freetype.js:1:20018 abort@http://localhost:8080/dist/html/freetype.js:24:10812 abortOnCannotGrowMemory@http://localhost:8080/dist/html/freetype.js:1:20872 enlargeMemory@http://localhost:8080/dist/html/freetype.js:1:21317 dynamicAlloc@http://localhost:8080/dist/html/freetype.js:1:6848 _sbrk@http://localhost:8080/dist/html/freetype.js:1:346073 Kp@http://localhost:8080/dist/html/freetype.js:7:105374 _w@http://localhost:8080/dist/:145:30 Xw@http://localhost:8080/dist/:130:68 ux@http://localhost:8080/dist/:9:396 L_e@http://localhost:8080/dist/:5:20 DXe@http://localhost:8080/dist/:150:93 td@http://localhost:8080/dist/:49:318 yd@http://localhost:8080/dist/:156:60 gh@http://localhost:8080/dist/:69:216 Bg@http://localhost:8080/dist/:1:21888 mg/<@http://localhost:8080/dist/:155:53  If this abort() is unexpected, build with -s ASSERTIONS=1 which can give more information.

After some research, I find the font file size > 10M(almost/around this value) will cause this problem. 
Do you have any idea on this? or this is a limit of javascrip side?
Thank you.

Here is ttf file for test (only, I copied from windows fonts folder). 
[Deng.zip](https://github.com/intrigus/gdx-freetype-gwt/files/2019598/Deng.zip)

**Comment by intrigus** [comment]
I'll look into this later. It should be easily fixable.

**Comment by intrigus** [comment]
@Vkrun could you please replace this [freetype.js](https://github.com/intrigus/gdx-freetype-gwt/blob/master/src/com/badlogic/gdx/graphics/g2d/freetype/public/freetype.js) with this [reetype.js](https://github.com/intrigus/gdx-freetype-gwt-gen/releases/tag/v.917b1e5ebe9d2116821325e690e55bfe1510667b) ?

**Comment by Vkrun** [comment]
@intrigus Thanks.
I have tried your new [freetype.js](https://github.com/intrigus/gdx-freetype-gwt-gen/releases/download/v.917b1e5ebe9d2116821325e690e55bfe1510667b/freetype.js) (size is 933 KB, the original one is 1.83 MB).
But no lucky, when I start the game, I got error message:
 `Uncaught Error: java.lang.RuntimeException: com.google.gwt.core.client.JavaScriptException: (TypeError) : $wnd.Module.writeArrayToMemory is not a function`

I just replace the dist/html/freetype.js in my game. Or maybe I should compile gdx-freetype-gwt from the source code?
